esi_leap/resource_objects/dummy_node.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, uuid):
     def get_resource_uuid(self):
         return self._uuid
 
-    def get_resource_name(self):
+    def get_resource_name(self, resource_list=None):
         return "dummy-node-%s" % self._uuid
```

Comment:
So we don't implement resource_list for dummy nodes?

Reply:
Nope. We need to query Ironic for ironic_nodes, but there's no need to query a name for dummy/test nodes. I was actually torn between putting an `if` statement in the controller to retrieve the name in this special way only if it's an ironic node, and doing it this way which allows the controller code to be slightly cleaner. I don't have a strong opinion one way or the other though!